src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
Yes, that was my [original suggestion](https://github.com/jquery/jquery/pull/2125/files#r26094668).